docs/stdlib/json.rst

```diff
@@ -182,21 +180,20 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
         db> select to_json('[1, "a", null]')[-1];
         {'null'}
 
-    The element access operator ``[]`` will raise an exception if the
-    specified index is not valid for the base JSON value.  To access
-    potentially out of bound indexes use the :eql:func:`json_get`
-    function.
+    This may raise an exception if the specified index is not valid for the
+    base JSON value. To access an element that is potentially out of
+    boundaries, use :eql:func:`json_get`.
 
 
 ----------
 
 
 .. eql:operator:: jsonslice: json [ int64 : int64 ] -> json
 
-    JSON array/string slicing.
+    Indexes an array or string of :eql:type:`json`.
 
-    JSON *arrays* and *strings* can be sliced in the same way as
-    regular arrays, producing a new JSON array or string:
+    This results in :eql:type:`json`. Arrays and strings of JSON can be sliced
+    in the same manner as an :eql:type:`array`, producing a JSON type:
```

Comment:
```suggestion
    Arrays and strings of JSON can be sliced in the same manner as an
    :eql:type:`array`, producing a :eql:type:`json` value:
```

Just discovered multi-line suggestions! 🎉 

Alright, the suggestion here is intended to reduce some redundancy. With the new one-liner suggestion, we now tell the user it produces a `json` value. We also reiterate that at the end of this paragraph. I suggest dropping the initial sentence which also says that. I also cleaned up the second sentence to clarify a value is produced from the slice, not a type.